### ISSUE TITLE
CA1711 Doesn't catch namespaces

### ISSUE THREAD

**Issue by jjhayworth** [dotnet/roslyn-analyzers#3353]
#### Analyzer package

[Microsoft.CodeAnalysis.FxCopAnalyzers](https://www.nuget.org/packages/Microsoft.CodeAnalysis.FxCopAnalyzers)

#### Package Version

v2.9.8 (Latest)

#### Diagnostic ID

Example: [CA1711](https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1711?view=vs-2019)

#### Repro steps

1. Create a new project, install FxCop Analyzers and configure to use AllRulesDefault.
2. Insert the following:

```
namespace ExistingIdentifier
{

}

namespace ExistingIdentifierEx
{

}

```

#### Expected behavior
Warning

#### Actual behavior
No warning.

**Comment by Evangelink** [comment]
I don't know if really makes sense to have this rule analyzing namespaces. @mavasani WDYT?

**Comment by mavasani** [comment]
@Evangelink Yes, the rule as implemented in legacy FxCop did not analyze namespaces, and our port to FxCop analyzers maintained this behavior. I think this can be closed by design and if we hear more feedback, we can re-open and consider this feature request again.
